src/keyring/KeyringController.test.ts

```diff
@@ -1102,4 +1135,319 @@ describe('KeyringController', () => {
       expect(cancelSignRequestStub.called).toBe(true);
     });
   });
+
+  describe('Ledger Keyring', () => {
+    let ledgerKeyring: LedgerKeyring;
+    let defaultAccount: any;
```

Comment:
Are you able to assign a type to this rather than `any`?

Reply:
[de405d3](https://github.com/MetaMask/controllers/pull/862/commits/de405d34d4267c613f0d3d68add2ff1a54f6ba46)